cmd/earth/main.go

```diff
@@ -1009,14 +1009,16 @@ func (app *earthApp) actionRegister(c *cli.Context) error {
 			fmt.Printf("%d) %s\n", i+1, key.String())
 		}
 		keyNum := promptInput("enter key number: ")
```

Comment:
I would also change this to

```suggestion
		keyNum := promptInput("enter key number (1=default): ")
```

Reply:
good catch, I lost this change when I rebased.